client/components/tinymce/plugins/media/restrict-size/index.js

```diff
@@ -58,6 +70,15 @@ export default function( editor ) {
 		}
 
 		event.content = setImages( event.content );
+
+		/**
+		 * Also add the raw resized image URL so we can properly preload it in the editor
+		 */
+		const resizedUrl = getResizedImgUrlFromImgString( event.content );
```

Comment:
Would you suggest a separate event/filter just to get the resized url, or to extract `BeforeSetWpcomMedia` to a separate event with a bit of duplicated code?